d3a_interface/read_user_profile.py

```diff
@@ -388,6 +388,9 @@ def read_profile_without_config(input_profile: Dict, slot_length_mins=15) -> Dic
 
 
 def read_and_convert_identity_profile_to_float(profile):
-    parsed_profile = ast.literal_eval(str(profile))
+    try:
+        parsed_profile = ast.literal_eval(str(profile))
```

Comment:
```suggestion
        parsed_profile = ast.literal_eval(profile)
```
I think explicit casting to str for literal_eval can be avoided here as the later is covered in exception raised as `ValueError `